oml/lightning/callbacks/metric.py

```diff
@@ -72,6 +89,30 @@ def on_validation_batch_end(
             if is_last_expected_batch:
                 self.calc_and_log_metrics(pl_module)
 
+    def _log_images(self, pl_module: pl.LightningDataModule) -> None:
+        if not isinstance(self.metric, EmbeddingMetrics):
+            return
+
+        for metric_name in flatten_dict(self.metric.metrics):  # type: ignore
+            if any([ignore in metric_name for ignore in self.image_metrics_to_ignore]):
+                continue
+            if self.log_only_main_category and not metric_name.startswith(OVERALL_CATEGORIES_KEY):
+                continue
+            fig = self.metric.get_plot_for_worst_queries(
+                metric_name=metric_name, topk_queries=LOG_TOPK_ROWS_PER_METRIC, topk_instances=LOG_TOPK_IMAGES_PER_ROW
+            )
+            metric_name_for_logging = metric_name.replace("/", "_")
+            log_str = f"{LOG_IMAGE_FOLDER}/epoch_{pl_module.current_epoch}/top {LOG_TOPK_ROWS_PER_METRIC} worst by {metric_name_for_logging}"
+            if isinstance(pl_module.logger, NeptuneLogger):
```

Comment:
Can we move to function `draw_fig_with_logger`? 

Why in case of `TensorBoardLogger` we use `current_epoch`, but with `NeptuneLogger` not?